pandas/sparse/series.py

```diff
@@ -570,27 +570,13 @@ def copy(self, deep=True):
         return self._constructor(new_data, sparse_index=self.sp_index,
                                  fill_value=self.fill_value).__finalize__(self)
 
+    @Appender(generic._shared_docs['reindex'] % _shared_doc_kwargs)
     def reindex(self, index=None, method=None, copy=True, limit=None,
                 **kwargs):
-        """
-        Conform SparseSeries to new Index
-
-        See Series.reindex docstring for general behavior
 
-        Returns
-        -------
-        reindexed : SparseSeries
-        """
```

Comment:
@toobaz you changed the API by allowing all of these other keywords. They were simply ignored before, now they are not. So obviously not tested. So they either need to be fully tested or raise NotImplementedError. So not ready to merge.